src/styles/_utility.scss

```diff
@@ -33,45 +33,75 @@ $scales-map: (
       3: 2,
       4: 2.5,
       5: 3,
+      6: 3.5,
+      7: 4,
+      8: 4.5,
+      9: 5,
+      10: 5.5,
     ),
     mobile: (
       1: 1,
-      2: 1.1,
```

Comment:
Those value were chosen deliberately because the flex layout breaks if we increase them.